docs/release-notes/february-2024.md

```diff
@@ -0,0 +1,129 @@
+---
+slug: february-2024
+hide_title: true
+description: >-
+    Release notes include the changes, fixes, and additions in specific versions of Semgrep.
+toc_max_heading_level: 3
+title: February 2024
+tags:
+  - Release notes
+---
+
+# Semgrep release notes for February 2024
+
+## 🔧 OSS Engine
+
+* The following versions of the OSS Engine were released in February 2024:
+  * [<i class="fas fa-external-link fa-xs"></i>1.59.1](https://github.com/semgrep/semgrep/releases/tag/v1.59.1)
+  * [<i class="fas fa-external-link fa-xs"></i>1.60.0](https://github.com/semgrep/semgrep/releases/tag/v1.60.0)
+  * [<i class="fas fa-external-link fa-xs"></i>1.60.1](https://github.com/semgrep/semgrep/releases/tag/v1.60.1)
+  * [<i class="fas fa-external-link fa-xs"></i>1.61.1](https://github.com/semgrep/semgrep/releases/tag/v1.61.1)
+  * [<i class="fas fa-external-link fa-xs"></i>1.62.0](https://github.com/semgrep/semgrep/releases/tag/v1.62.0)
+  * [<i class="fas fa-external-link fa-xs"></i>1.63.0](https://github.com/semgrep/semgrep/releases/tag/v1.63.0)
+
+## 🌐 Cloud Platform
+
+### Added
+
+### Changed
+
+### Fixed
+
+## 💻 Code
+
+### Added
+
+* Added new rules for Elixir and the Phoenix framework, covering various security and correctness issues. These are available in the `p/elixir`
+  ruleset. <!-- Do we need to mention that this is for Pro users only? -->
```

Comment:
Are these Semgrep Pro rules?

Reply:
Yup, unless I misunderstand this?

> To use them, users must be logged in and use the Pro engine via the --pro option! Take that, Semgrep OSS!

[Slack](https://semgrepinc.slack.com/archives/C03A5BETNS3/p1706885627454809)

Reply:
Since they are Pro rules, I think we can just say "Added new Pro rules..." I'll make the suggestion.